docs/types/Store.prototype.createData.md

```diff
@@ -3,8 +3,38 @@
 
 @signature `Store.prototype.createData(request, response)`
```

Comment:
There is no description.  Please make sure all pages have a description.

Reply:
Do we write out the `Store.prototype` part in any other methods?

At: http://localhost:8080/canjs/doc/can-define/map/map.prototype.assign.html

We show `map.assign(props)`

This should probably be `store.createData(request, response)`.  Please update other examples.

Reply:
Removed prototype from all method signatures. 487d7bf